### PR TITLE
add tests for RI-4591

### PR DESCRIPTION
**Pull request by mariasergeenko**

### CODE REVIEW

File: tests/e2e/pageObjects/triggers-and-functions-page.ts [hunk 1]
```diff
@@ -27,5 +34,41 @@ export class TriggersAndFunctionsPage extends InstancePage {
 
         return item;
     }
-}
 
+    /**
+     * Is function displayed in the list
+     * @param functionsName The functions Name
+     * @param sectionName The section Name
```
Comment (vlad-dargel) [inline tests/e2e/pageObjects/triggers-and-functions-page.ts:41]: wrong params ordering here

File: tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts [hunk 2]
```diff
@@ -41,3 +41,61 @@ test
         await t.expect(row.totalFunctions).eql(item.totalFunctions, 'user name is unexpected');
     });
 
+test.only
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts:44]: .only

File: tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts [hunk 3]
```diff
@@ -41,3 +41,61 @@ test
...
+        await t.click(await triggersAndFunctionsPage.getLibraryNameSelector(libraryName));
+        await t.expect(await triggersAndFunctionsPage.getFunctionsByName(LibrariesSections.Functions, function1).exists).ok('library is not displayed in Functions section');
+        await t.expect(await triggersAndFunctionsPage.getFunctionsByName(LibrariesSections.Functions, function2).exists).ok('library is not displayed in Functions section');
+        await t.expect(await triggersAndFunctionsPage.getFunctionsByName(LibrariesSections.Functions, asyncFunction).exists).ok('library is not displayed in Functions section');
+        await t.expect(await triggersAndFunctionsPage.getFunctionsByName(LibrariesSections.StreamFunctions, stream).exists).ok('library is not displayed in Stream section');
+        await t.expect(await triggersAndFunctionsPage.getFunctionsByName(LibrariesSections.ClusterFunctions, cluster).exists).ok('library is not displayed in cluster section');
+        await t.expect(await triggersAndFunctionsPage.getFunctionsByName(LibrariesSections.KeyspaceTriggers, keySpaceTrigger).exists).ok('library is not displayed in key Space Trigger section');
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts:73]: maybe it's better to make a cycle here?

File: tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts [hunk 4]
```diff
@@ -41,3 +41,61 @@ test
         await t.expect(row.totalFunctions).eql(item.totalFunctions, 'user name is unexpected');
     });
 
+test.only
+    .after(async() => {
+        await browserPage.Cli.sendCommandInCli(`TFUNCTION DELETE ${libraryName}`);
+        await deleteStandaloneDatabaseApi(ossStandaloneRedisGears);
+    })('Verify that library details is displayed', async t => {
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts:48]: do you have these tests names into checklist?

Comment (mariasergeenko) [reply tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts:48]: not yet, I usually updated it after merge and will ask Roma and Amir mark their unit tests

**Review by vlad-dargel — APPROVED** [review]

**Pull request merged**